src/Sketch.java

```diff
@@ -1,24 +1,63 @@
 import processing.core.PApplet;
 
+/**
```

Comment:
Great comment, very helpful! Impressive project.

tiny nitpick: You should avoid using the `/** */` documentation comment structure for comments like this that are intended for teachers or other programmers. Use `/* */` for that. documentation comments are special and their intended audience is people using the code abstractly.